### ISSUE TITLE
Make resources work after extraction

### ISSUE THREAD

**Issue by piegamesde** [gildas-lormeau/SingleFileZ#141]
I really like the concept of having all resources bundled in a zip file, as opposed to having to embed them as base64 in the html. However, it's a bit unfortunate that the extracted output only works as zip file: when extracting the contents and opening it, all resources are broken. It'd be cool to have both. The idea is that JavaScript is only required for the single-file, and one still has a fully static archive if needed.

I don't know much about how this works internally, but for me it'd be fine if the zip archive contained a separate `index_archive.html` that does this job instead of having everything within a single `index.html`. (Or maybe swap the names, and have `index_singlefile.html` instead)

(Related to #102 and #119)

**Comment by gildas-lormeau** [comment]
> all resources are broken

What do you mean?

**Comment by piegamesde** [comment]
Downloaded page with SingleFileZ:

![image](https://user-images.githubusercontent.com/14054505/204094483-fe5bec0d-27a0-40a5-83db-4847024648bb.png)

Renamed the file to .zip, extracted it into a folder and opening its `index.html`:

![image](https://user-images.githubusercontent.com/14054505/204094527-d312524e-400c-4e84-9686-e66a3646c773.png)

**Comment by gildas-lormeau** [comment]
It's limitation of the browser because you're opening the page from the filesystem. If you use an HTTP server, resources will be displayed as expected.

**Comment by piegamesde** [comment]
Oh, indeed. I'm sorry for the confusion then, I thought this limitation only applied to the single-file archive and not to the extracted contents.

**Comment by gildas-lormeau** [comment]
Note that it used to work in the past, but vendors make the use of the filesystem less and less convenient, probably for security reasons.

**Comment by piegamesde** [comment]
I see. Then the best thing we can do is to warn users about this better, not sure how though. Maybe add a README to the archive that hints at [miniserve](https://github.com/svenstaro/miniserve) or something like that?

**Comment by gildas-lormeau** [comment]
I agree, I'll try to document this issue. I'll need to tests it in all browsers.
